Sources/Controllers/Map/Layers/OAPOILayer.mm

```diff
@@ -479,6 +479,7 @@ - (void) collectObjectsFromPoint:(CLLocationCoordinate2D)point touchPoint:(CGPoi
             }
             else
             {
+                poi.isRenderedObject = YES;
```

Comment:
Do not looks right. Why `obfsDataInterface->findAmenityByObfMapObject `did not find amenity?